src/spikeinterface/core/sparsity.py

```diff
@@ -11,29 +11,29 @@
 
 
 _sparsity_doc = """
-    method: str
-        * "best_channels": N best channels with the largest amplitude. Use the "num_channels" argument to specify the
+    method : str
```

Comment:
Since this is an injected docstring does the numpydoc validation still work the same?